src/main/java/athena/Parser.java

```diff
@@ -14,239 +14,119 @@
     public static final String IMPORTANCE_DELIMITER = "i/";
     public static final String ADDITIONAL_NOTES_DELIMITER = "a/";
 
+    /**
+     * Get Parameters description for Add command.
+     * @param taskInformation String representing task information
+     * @param delimiter String representing parameter delimiter
+     * @return Description of parameter
+     */
+    public static String getParameterForAdd(String taskInformation, String delimiter) {
+        String param;
+        String[] retrieveParamInfo = taskInformation.split(delimiter, 2);
+        String retrievedParamInfo = retrieveParamInfo[1];
+        int paramNextSlash = retrievedParamInfo.indexOf("/");
+        if (paramNextSlash == -1) {
+            param = retrievedParamInfo;
+        } else {
+            param = retrievedParamInfo.substring(0, (paramNextSlash - 2));
+        }
+        return param;
+    }
+
+    /**
+     * Get Parameters description for Edit and List command.
+     * @param taskInformation String representing task information
+     * @param delimiter String representing parameter delimiter
+     * @param paramPosition Integer representing position of parameter
+     * @return Description of parameter
+     */
+    public static String getParameterForEditAndList(String taskInformation, String delimiter, int paramPosition) {
+        String param;
+        if (paramPosition == -1) {
+            param = "";
+        } else {
+            String[] retrieveParamInfo = taskInformation.split(delimiter, 2);
+            String retrievedParamInfo = retrieveParamInfo[1];
+            int paramNextSlash = retrievedParamInfo.indexOf("/");
+            if (paramNextSlash == -1) {
+                param = retrievedParamInfo;
+            } else {
+                param = retrievedParamInfo.substring(0, (paramNextSlash - 2));
+            }
+        }
+        return param;
+    }
```

Comment:
These 2 methods look like they are doing almost the same thing. Can we combine them together?